### DISCUSSION TITLE
Permisssion Delegation and Revoke Delegated Permission

### DISCUSSION THREAD

**Discussion by KabeerHamza** [spatie/laravel-permission#2246] [Ideas]
Preamble,

One thing I am yet to see in Laravel Spatie is issue that has to do with **Permisssion Delegation and Revoke Delegated Permission**,  In some instances a user will will be require to have additional permission from a given role, You dont want to assign a direct role as such but some permission under given role.

I hope you get the new features

MOHAMMED | mohdkabr@gmail.com

**Comment by drbyte** [comment]
You can do that by giving **direct** permission to the `user` with `$user->givePermissionTo('edit articles');`
(instead of only giving permission to the `role`)
